docs/en/get_started/quick_start.md

```diff
@@ -0,0 +1,745 @@
+# Quick Start
+
+This chapter will introduce you to all the syntax and basic concepts of Ribir.
+
+> You will learn
+>
+> - How to create and compose widgets
+> - How to respond to events and operate data
+> - How to make the view automatically respond to data changes
+> - How to build dynamic widgets
+> - How to map your own data structure to a view
+> - How to use built-in widgets as part of other widgets
+> - How to convert, separate and trace original state -- to facilitate the transfer of state and control the scope of view updates
+
+## What is a widget?
+
+In Ribir, the widget is the basic unit for describing the view. In form, it can be a button, a text, a list, a dialog, or even the entire application interface. In code, it can be a function, a closure, or a data object. The type that Ribir can build through `&BuildCtx` is called widget. Note the difference between `Widget` and widget, in the context of the entire Ribir, widget is a generic term, and the capitalized `Widget` is a specific widget, which is also the pass for all widgets to enter the view.
+
+If you don't understand the above words very well, don't worry, because you don't need to care about the construction process of the widget at all, and Ribir also prohibits interference in this process. You only need to understand that Ribir divides all widgets into four categories:
```

Comment:
`Ribir also prohibits interference in this process` -> `Ribir also prohibits developer interference in this process`.